.pipelines/Common.yml

```diff
@@ -0,0 +1,121 @@
+parameters:
+  - name: official
+    type: boolean
+
+stages:
+- stage: build
+  jobs:
+  - job: bicep_windows
```

Comment:
In normal ADO the job fanout can be done with matrix just like in GH actions. I'm not sure if matrix works with the "extends" syntax, it doesn't seem like it'll gain us much, so I didn't try it out yet.